src/components/createContainer.jsx

```diff
@@ -38,109 +42,152 @@ export default function createContainer(DecoratedComponent, specs) {
 
     static contextTypes = {
       adrenaline: PropTypes.shape({
-          renderLoading: PropTypes.func.isRequired,
-          adaptor: adaptorShape.isRequired
-      }).isRequired
+        renderLoading: PropTypes.func.isRequired,
+        adaptor: adaptorShape.isRequired,
+      }).isRequired,
     }
 
     static childContextTypes = {
-        adrenaline: containerShape.isRequired
+      adrenaline: containerShape.isRequired,
     }
 
     constructor(props, context) {
       super(props, context);
-      this.isNotMounted = true;
-      this.state = {};
+      this.state = {
+        current: null, // The currently committed slice of data,args,queries
```

Comment:
In order to increase code maintainability lets try to use more descriptive variable names.

Here we can use `commitedState`.